ModuleFile

```diff
@@ -1,7 +1,8 @@
 name 'citops-repose'
-version '1.8.2'
+version '2.0.0-1'
 description "Repose is an api middleware that provides authentication,
 filtering, ratelimitting and several other features, this deploys it."
 project_page 'https://github.com/rackerlabs/puppet-repose'
 
 dependency 'puppetlabs/stdlib', '>= 2.0.0'
+dependency 'justin8/systemd', '>= 0.4.0'
```

Comment:
How is camptocamp's module working in your p4 now @bjoshua ?

https://github.com/camptocamp/puppet-systemd/pull/18 might be an issue.

Also, to @mwhahaha 's point the systemd::service_limits would work if you pass your contents via `$contents`.  I'm not excited about that interface because it feels kinda like a misuse or poorly implemented pattern.